techniques/css/C43.html

```diff
@@ -268,29 +268,29 @@ <h2>Related Resources</h2>
 
 
 <section id="related">
-			<h2>Related Techniques</h2>
+			<h2>関連テクニック</h2>
 			<ol>
 				<li><a href="../css/C32">C32: Using media queries and grid CSS to reflow columns</a></li>
 				<li><a href="../css/C34">C34: Using media queries to un-fixing sticky headers / footers</a></li>
 			</ol>
 
 		</section>
 <section id="tests">
-			<h2>Tests</h2>
+			<h2>検証</h2>
 			<section class="test-procedure" id="procedure">
-				<h3>Procedure</h3>
+				<h3>手順</h3>
 
-				<p>For each user interface component that can receive keyboard focus:</p>
+				<p>キーボードフォーカスを受け取ることができる各ユーザインタフェース コンポーネントについて:</p>
 				<ol>
-					<li>Check that the user interface component is not entirely hidden when it receives keyboard focus.</li>
-          <li>Check that no part of the user interface component is hidden when it receives keyboard focus.</li>
+					<li>キーボードフォーカスを受け取ったときに、ユーザインタフェース コンポーネントが完全に隠れていないことをチェックする。</li>
+          <li>キーボードフォーカスを受け取ったときに、ユーザインタフェース コンポーネントのどの部分も隠れていないことをチェックする。</li>
 				</ol>
 			</section>
 			<section class="test-results" id="expected-results">
-				<h3>Expected Results</h3>
+				<h3>期待される結果</h3>
 				<ul>
-					<li>Check #1 is true.</li>
-          <li>For Focus Not Obscured (Enhanced) (Level AAA), #2 is also true.</li>
+					<li>1 が真である。</li>
```

Comment:
ここの表現ですが、既存の他のテクニック (WCAG 2.1 達成方法集) では「#1 の結果が真である。」のような表現になっています。
```suggestion
					<li>#1 の結果が真である。</li>
```